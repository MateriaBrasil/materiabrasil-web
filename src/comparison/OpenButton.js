import React from 'react'
import Button from '@material-ui/core/Button'
import Icon from '@material-ui/icons/ExpandLess'

export default props => (
  <Button variant="contained" color="primary" {...props}>
    Comparação de materiais
    <Icon style={{ marginLeft: 8 }} />
  </Button>
)
