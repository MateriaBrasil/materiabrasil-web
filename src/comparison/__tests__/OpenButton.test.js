import React from 'react'
import renderer from 'react-test-renderer'

import OpenButton from '../OpenButton'

it('renders correctly', () => {
  const tree = renderer.create(<OpenButton foobar={'foo'} />).toJSON()
  expect(tree).toMatchSnapshot()
})
