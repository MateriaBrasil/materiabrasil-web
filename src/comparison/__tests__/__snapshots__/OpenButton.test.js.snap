// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedPrimary-13 MuiButton-raised-15 MuiButton-raisedPrimary-16"
  disabled={false}
  foobar="foo"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    Comparação de materiais
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-30"
      color={undefined}
      focusable="false"
      role="presentation"
      style={
        Object {
          "marginLeft": 8,
        }
      }
      viewBox="0 0 24 24"
    >
      <path
        d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"
      />
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
    </svg>
  </span>
  <span
    className="MuiTouchRipple-root-39"
  />
</button>
`;
